### PR TITLE
web/satellite added new billing screen feature flag

### PR DESCRIPTION
**Pull request by cl-mitch**
What: Added a new feature flag for upcoming billing screen work.

Why: The new flag is used to enable the new billing flow in the satellite UI.

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/console/consoleweb/server.go [hunk 1]
```diff
@@ -96,6 +96,7 @@ type Config struct {
 	NewNavigation                   bool    `help:"indicates if new navigation structure should be rendered" default:"true"`
 	NewObjectsFlow                  bool    `help:"indicates if new objects flow should be used" default:"false"`
 	NewAccessGrantFlow              bool    `help:"indicates if new access grant flow should be used" default:"false"`
+	NewBillingScreen                bool    `help:"indicates if new access grant flow should be used" default:"false"`
```
Comment (VitaliiShpital) [inline satellite/console/consoleweb/server.go:99]: seems like help info is incorrect

File: scripts/testdata/satellite-config.yaml.lock [hunk 2]
```diff
@@ -241,6 +241,9 @@ compensation.withheld-percents: 75,75,75,50,50,50,25,25,25,0,0,0,0,0,0
 # indicates if new access grant flow should be used
 # console.new-access-grant-flow: false
 
+# indicates if new billing screen should be used
```
Comment (VitaliiShpital) [inline scripts/testdata/satellite-config.yaml.lock:244]: did you add it manually? Config lock test is failing so please try to run `go generate ./satellite/...` from root folder and revert all the unnecessary changes that will appear (except changes for config-lock file)

File: satellite/console/consoleweb/server.go (hunk 1 above)
Comment (cl-mitch) [reply satellite/console/consoleweb/server.go:99]: fixed

File: scripts/testdata/satellite-config.yaml.lock (hunk 2 above)
Comment (cl-mitch) [reply scripts/testdata/satellite-config.yaml.lock:244]: re-implemented using `go generate ./satellite` from root

**Review by VitaliiShpital — APPROVED** [review]

**Review by mobyvb — APPROVED** [review]

**Review by NikolaiYurchenko — APPROVED** [review]

**Pull request merged**